.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md

```diff
@@ -1,73 +1,50 @@
-**This pull request is opened against a release branch.<br>
-To request a cherry pick of a commit, please fill in the form below.** (Questions with an asterisk are required.)<br>
+This pull request is created by [automatic cherry pick workflow](https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process#automatically-creates-a-cherry-pick-request)
+Please fill in the form below, and a flutter domain expert will evaluate this cherry pick request.
 
-**To fill in the form, you can edit this PR description and type your answers after the 'My Answer' keywords. <br>
-A flutter domain expert will evaluate this cherry pick request shortly after ALL questions are answered.**
+---
+#### Issue Link:
+What is the link to the issue this cherry-pick is addressing?<br>
```

Comment:
Are breaks `<br>` necessary for line breaks here?

Reply:
Good catch! removed